libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -278,142 +299,149 @@ export class XCCDFResultsMapper extends BaseConverter {
         status: 'loaded',
         controls: [
           {
-            path: 'Benchmark.Group',
+            path: 'Benchmark',
+            pathTransform: getRulesInBenchmark,
             key: 'id',
             tags: {
               cci: {
-                path: 'Rule.ident',
+                path: ['ident', 'reference'],
                 transformer: extractCci
               },
               nist: {
-                path: 'Rule.ident',
+                path: ['ident', 'reference'],
                 transformer: nistTag
               },
-              severity: {path: 'Rule.severity'},
+              severity: {path: 'severity'},
               description: {
-                path: ['Rule.description.text', 'Rule.description'],
-                transformer: (description: string) =>
-                  JSON.stringify(
-                    _.pickBy(convertEncodedXmlIntoJson(description), _.identity)
-                  )
+                path: ['description.text', 'description'],
+                transformer: (description: string): string =>
+                  _.get(
+                    parseXml(description),
+                    'VulnDiscussion',
+                    description
+                  ) as string
               },
-              group_id: {path: 'id'},
-              group_title: {path: ['title.text', 'title']},
+              group_id: {path: 'group.id'},
```

Comment:
![image](https://github.com/mitre/heimdall2/assets/2015821/e352b6a6-b743-48b8-af45-69872335b5a6)
<img width="490" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/32750567-6345-437d-bf37-9aea3ea12c6e">

doubly nested groups only seem to show the immediate parent group's information.  i feel like all levels of group information should be shown here, but am not sure how best to reflect that information such that it shows up well within heimdall.  the also showcases other problems like how certain group level attributes don't get surfaced properly such as that 'value' block.  probably worthwhile setting up a brief call with you/me/aaron/eugene to figure out how to handle this one.

Reply:
making this a separate issue to figure out a good solution for multiply-nested groups.  for now, only showing the parent group is sufficient.